docs/docsite/rst/dev_guide/developing_program_flow_modules.rst

```diff
@@ -378,79 +378,75 @@ Internal arguments
+
+You can use this argument to control the level (0 for none) of verbosity in logging. 
+
 
 _ansible_selinux_special_fs
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-List. Names of filesystems which should have a special SELinux
-context. They are used by the `AnsibleModule` methods which operate on
-files (changing attributes, moving, and copying). To set, add a comma separated string of filesystem names in :file:`ansible.cfg`:
+Type: ``list``
+Elements: ``strings``
+
+This argument provides modules with the names of file systems which should have a special SELinux context. They are used by the ``AnsibleModule`` methods which operate on files (changing attributes, moving, and copying). To set, add a comma-separated string of file system names in the :file:`ansible.cfg` configuration file:
 
 .. code-block:: ini
 
   # ansible.cfg
   [selinux]
   special_context_filesystems=nfs,vboxsf,fuse,ramfs,vfat
 
-Most modules can use the built-in ``AnsibleModule`` methods to manipulate
-files. To access in a module that needs to know about these special context filesystems, instantiate an ``AnsibleModule`` and examine the list in
-:attr:`AnsibleModule._selinux_special_fs`.
+Most modules can use the built-in ``AnsibleModule`` methods to manipulate files. To access in a module that needs to know about these special context file systems, instantiate ``AnsibleModule`` and examine the list in :attr:`AnsibleModule._selinux_special_fs`.
 
-This replaces :attr:`ansible.module_utils.basic.SELINUX_SPECIAL_FS` from
-:ref:`module_replacer`. In module replacer it was a comma separated string of
-filesystem names. Under Ansiballz it's an actual list.
+This argument replaces :attr:`ansible.module_utils.basic.SELINUX_SPECIAL_FS` from :ref:`module_replacer`. In the module replacer framework the argument was formatted as a comma-separated string of file system names. Under the Ansiballz framework it is a list.
 
 .. versionadded:: 2.1
 
+
 _ansible_syslog_facility
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-This parameter controls which syslog facility Ansible module logs to. To set, change the ``syslog_facility`` value in :file:`ansible.cfg`. Most
-modules should just use :meth:`AnsibleModule.log` which will then make use of
-this. If a module has to use this on its own, it should instantiate an
-`AnsibleModule` and then retrieve the name of the syslog facility from
-:attr:`AnsibleModule._syslog_facility`. The Ansiballz code is less hacky than the old :ref:`module_replacer` code:
+This argument controls which syslog facility the module logs to. To set, change the ``syslog_facility`` value in the :file:`ansible.cfg` configuration file. Most modules should just use the :meth:`AnsibleModule.log` function which will then make use of this. If a module has to use this on its own, it should instantiate the ``AnsibleModule`` method and then retrieve the name of the syslog facility from :attr:`AnsibleModule._syslog_facility`. The Ansiballz code is less elegant than the :ref:`module_replacer` code:
```

Comment:
Instead of mentioning one way to set the syslog facility using the ansible.cfg file, this would be another good place to reference the config option instead https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-syslog-facility since it is more comprehensive and the documentation won't become out of sync as easily.

Reply:
I think "To set, change the ``syslog_facility`` value in the :file:`ansible.cfg` configuration file." should be removed. Listing individual ways to set these internal parameters means we need to maintain those here and in the config file. I'm not going to comment on each internal parameter, but this is directed at any internal parameter still listing individual ways to set it (besides keywords, as there isn't a way to link to an individual playbook keyword, that I know of).

It's difficult to keep developer documentation in sync, because people already familiar with the infrastructure aren't regularly reading it. Keeping these explanations general and pointing to the source of truth will let this page stay accurate and complete more easily.

Reply:
Ok, i attempted to remove those individual ways. Hopefully, I covered all.